bin.src/make_remote_butler.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
+# This file is part of prompt_prototype.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+
+"""Simple script for creating a repository at a remote URI, given
+a source repository and export file.
+
+For most values of --target-repo and --seed-config, this script is only useful
+if run from the prompt-proto project on Google Cloud.
+
+The user is responsible for clearing any old copies of the repository from
+both the target URI and the registry database.
+"""
+
+
+import argparse
+import logging
+import os
+import sys
+
+import lsst.log
```

Comment:
No butler code should be issuing lsst.log messages so please do not add lsst.log dependencies in code like this. In fact no python code should be using lsst.log and if there is something that is importing some code that sets the lsst.log logger we can talk about the best way to do it in lsst.utils.logging or using the butler code without making lsst.log visible anywhere.

Reply:
This is simply vestigial code; there's no reason to get so worked up about it.